satellite/satellitedb/utils.go

```diff
@@ -19,3 +23,45 @@ func bytesToUUID(data []byte) (uuid.UUID, error) {
 
 	return id, nil
 }
+
+type nodeIDsArray storj.NodeIDList
+
+// Value converts a NodeIDList to a postgres array
+func (nodes nodeIDsArray) Value() (driver.Value, error) {
```

Comment:
I love Valuers, but in this case I feel like we're enshrining database-specific serialization logic into generic Go code.

Reply:
This is the reason it's not a method on `NodeIDList` type itself, but rather a separate type with the method. Maybe it might make sense to include `postgres` in the name?

Reply:
Ah yeah, I didn't catch that.  I like having postgres in the name